File: Assignment 2/if else/evenodd.cpp
#include<iostream>
using namespace std;
class Even_Odd
{
    int number;
    public: void evenodd()
    {
        cout<<"Enter a number:\n";
        cin>>number;
        if(number%2==0)
        {
            cout<<"The number is even";
        }
        else
        {
            cout<<"The number is odd";
        }
    }
};
int main()
{
    Even_Odd obj;
    obj.evenodd();
}

File: Java assignment(1-2)/java Assignment 2/Even_Odd.java
import java.util.Scanner;
class Even_Odd
{
int num1;
void getData()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter a positive integer: ");
num1=sc.nextInt();
if(num1%2==0)
{
System.out.println("The integer is even");
}
else
{
System.out.println("The integer is odd");
}
}
public static void main(String...args)
{
Even_Odd obj=new Even_Odd();
obj.getData();
}
}